libs/cardano-ledger-core/src/Cardano/Ledger/PoolParams.hs

```diff
@@ -191,7 +191,7 @@ instance DecCBOR StakePoolRelay where
       k -> invalidKey k
 
 -- | A stake pool.
-data PoolParams = PoolParams
+data StakePoolParams = StakePoolParams
```

Comment:
all member should change such that instead of starting with `pp` , start with `sp`, yes?